components/public-api/typescript-common/src/public-api-converter.ts

```diff
@@ -1211,6 +1213,41 @@ export class PublicAPIConverter {
                 );
             }
         }
+        const jetbrainsIdes = prebuild.workspace.config.jetbrains;
```

Comment:
please add a comment that it is HACK, we starting to duplicating and hardcoding logic of JetBrains in server